### DISCUSSION TITLE
May layouts be nested?

### DISCUSSION THREAD

**Discussion by ghost** [Megabit/Blazorise#3366] [Q&A]
May layouts be nested?
Like it is shown on https://blazorise.com/docs/components/layout
it IS nested, but this seems strange to me.
`<Layout>
    <LayoutHeader Fixed="true">
        Header
    </LayoutHeader>
    <Layout Sider="true">
        <LayoutSider>
            <LayoutSiderContent>
                Sider
            </LayoutSiderContent>
        </LayoutSider>
        <Layout>
            <LayoutContent>
                Content
            </LayoutContent>
        </Layout>
    </Layout>
</Layout>`
What I don't understand is why I still need nested Layout-Tags here, even if I'm using LayoutSider, ...Content, ...Header and so on. 
Unfortunately these additional tags (...Sider etc) aren't described in detail on the docu page).

**Comment by stsrki** [comment]
Yes, layouts can be nested. The order and structure of the `Layout` can lead to different layouts where the sidebar and topbar, for example, can be structured. As you can already see in our examples.

**Comment by ghost** [comment]
> As you can already see in our examples.
Not really. I tried the following example
![grafik](https://user-images.githubusercontent.com/10619439/151433442-35f74e11-be9b-446b-8618-13144862b5e0.png)
But in my case it looks:
![grafik](https://user-images.githubusercontent.com/10619439/151433490-e8705a24-6213-454e-8f9e-a6759bff7261.png)
Here's my code:
![grafik](https://user-images.githubusercontent.com/10619439/151433721-3b9c89da-a2d1-467d-b84a-666d801f71ef.png)
